extensions/wikia/UserLogin/UserLoginHooksHelper.class.php

```diff
@@ -116,7 +116,12 @@ public static function onSetUserEmail( User $user, $newEmail, &$result, &$info )
 	}
 
 	public static function isValidEmailAddr( $addr ) {
-		return preg_match( '/^[a-z0-9._%+-]+@(?:[a-z0-9\-]+\.)+[a-z]{2,4}$/i', $addr ) !== 0;
+		// this validation match Helios email validation
+		// [see: https://github.com/Wikia/helios/blob/ce490394ae10eb248afd2a063dc465d8eb4efb01/domain/user_validator_funs.go#L29]
+		return preg_match(
+			'/^[\w!#$%&\'*+\/=?^_`{|}~-]+(?:\.[\w!#$%&\'*+\/=?^_`{|}~-]+)*@(?:[\w](?:[\w-]*[\w])?\.)+[a-zA-Z0-9](?:[\w-]*[\w])?$/i',
+			$addr
+		) !== 0;
```

Comment:
Though, it seems that filter_var doesn't handle multibyte emails either.  Given neither supports this, I'd rather go with the built in `filter_var` for now and then address this when the Helios team expands to support multibyte.